tests/shared/src/test/scala/cats/tests/IorSuite.scala

```diff
@@ -299,6 +299,21 @@ class IorSuite extends CatsSuite {
     assert(ior.toIorNec === (Ior.both[NonEmptyChain[String], Int](NonEmptyChain.one("oops"), 42)))
   }
 
+  test("toIorA Left") {
+    val ior = Ior.left[String, Int]("oops")
+    assert(ior.toIorA[NonEmptyVector, String] === (Ior.left[NonEmptyVector[String], Int](NonEmptyVector.one("oops"))))
+  }
+  test("toIorA Right") {
+    val ior = Ior.right[String, Int](42)
+    assert(ior.toIorA[NonEmptyVector, String] === (Ior.right[NonEmptyVector[String], Int](42)))
+  }
+  test("toIorA Both") {
+    val ior = Ior.both[String, Int]("oops", 42)
+    assert(
+      ior.toIorA[NonEmptyVector, String] === (Ior.both[NonEmptyVector[String], Int](NonEmptyVector.one("oops"), 42))
+    )
+  }
```

Comment:
Can we get rid of hard-coded test values here as well please?

Reply:
sure, haven't noticed, sorry